packages/utilities/src/Async.ts

```diff
@@ -49,7 +49,7 @@ export class Async {
     if (this._immediateIds) {
       for (id in this._immediateIds) {
         if (this._immediateIds.hasOwnProperty(id)) {
-          this.clearImmediate(id);
+          this.clearImmediate(parseInt(id, 10));
```

Comment:
I don't think that's a bug, unless it's browser specific. I just tried it by caching the stringified value of the response of setTimeout and then calling clearTimeout with it and it worked. aRE YOU SEEING AN ACTUAL BUG?